user-guide/Advanced_Modules/Swarming/Messages/ElementUnavailableWhileSwarming.md

```diff
@@ -0,0 +1,11 @@
+---
+uid: ElementUnavailableWhileSwarming
+---
+
+# Message: "x" is currently switching
```

Comment:
would change x to something else, maybe elementName or something

Reply:
updated